ALUM/ALUM/Models/ProfileModel.swift

```diff
@@ -8,11 +8,6 @@
 import Foundation
 import SwiftUI
 
-struct MentorGetData: Decodable {
```

Comment:
MentorGetData is something that doesn't belong to the ProfileModel but UserService so moved it there